pollination/alias/inputs/bool_options.py

```diff
@@ -0,0 +1,56 @@
+from pollination_dsl.alias import InputAlias
+from queenbee.io.common import IOAliasHandler
+
+
+"""Alias for yes/no inputs about whether to filter design days."""
+filter_des_days_input = [
+    InputAlias.any(
+        name='filter_des_days',
+        description='Either a boolean or one of two text strings: filter-des-days, '
+        'all-des-days.',
+        platform=['grasshopper'],
+        handler=[
+            IOAliasHandler(
+                language='python',
+                module='pollination_handlers.inputs.bool_options',
+                function='filter_des_days_to_str'
+            )
+        ]
+    )
+]
+
+
+"""Alias for yes/no inputs about whether to use multipliers."""
+use_multiplier_input = [
+    InputAlias.any(
+        name='use_multiplier',
+        description='Either a boolean or one of two text strings: multiplier, '
+        'full-geometry.',
+        platform=['grasshopper'],
+        handler=[
+            IOAliasHandler(
+                language='python',
+                module='pollination_handlers.inputs.bool_options',
+                function='use_multiplier_to_str'
+            )
+        ]
+    )
+]
+
+
+"""Alias for yes/no inputs about whether a building is residential."""
+use_multiplier_input = [
```

Comment:
You need to change the name here otherwise it will overwrite the one for using multipliers.

Reply:
Good catch!